pyproject.toml

```diff
@@ -25,6 +25,7 @@ requires-python = ">=3.8"
 # For an analysis of this field vs pip's requirements files see:
 # https://packaging.python.org/discussions/install-requires-vs-requirements/
 dependencies = [
+    "aesera",
```

Comment:
aesara shouldn't be required, we changed the backend to pytensor in pymc 5.0.

Reply:
It was suspicious to me as well, but couldn't figure out why the tests failed with `ERROR  - ModuleNotFoundError: No module named 'aesara'`. I think pymc_bart must be causing this error. Could it be because I did not specify the version requirement for it?

Reply:
Is it not `pymc-bart` with a dash and not an underscore?